staging/src/k8s.io/kubectl/pkg/describe/describe.go

```diff
@@ -746,7 +747,7 @@ func (d *PodDescriber) Describe(namespace, name string, describerSettings Descri
 	var events *corev1.EventList
 	if describerSettings.ShowEvents {
 		if ref, err := reference.GetReference(scheme.Scheme, pod); err != nil {
-			klog.Errorf("Unable to construct reference to '%#v': %v", pod, err)
+			klog.ErrorS(err, "Unable to construct reference to", pod)
```

Comment:
Is this a user-visible change? If it is, we should update the changelog entry from `NONE`.

Reply:
Done